ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ServiceToOpenAPIMapper.java

```diff
@@ -80,6 +83,23 @@ public static List<OASResult> generateOAS3Definition(Project project, SyntaxTree
                                                          SemanticModel semanticModel,
                                                          String serviceName, Boolean needJson,
                                                          Path inputPath) {
+        return generateOAS3Definition(project, syntaxTree, semanticModel, serviceName, needJson, inputPath, "0");
```

Comment:
Rather than hardcoding `0` here, can't we make it a constant like `DEFAULT_EXTENSION_LEVEL` and use it here ?

Reply:
Fixed via [5e382c9](https://github.com/ballerina-platform/openapi-tools/pull/1745/commits/5e382c9d1336b5cfed22a3ebfb0618b14f328a15)